### DISCUSSION TITLE
Using a different logo for your team.

### DISCUSSION THREAD

**Discussion by riffnshred** [riffnshred/nhl-led-scoreboard#310] [Dev and Ideas]
It was requested on discord to get a reminder on how you can use different logos for your team so here it is.

[Here is a link to all the logos available on the Nhl API. ](https://records.nhl.com/site/api/franchise?include=teams.id&include=teams.active&include=teams.triCode&include=teams.placeName&include=teams.commonName&include=teams.fullName&include=teams.logos&include=teams.conference.name&include=teams.division.name&include=teams.franchiseTeam.firstSeason.id&include=teams.franchiseTeam.lastSeason.id&include=teams.franchiseTeam.teamCommonName)

Simply search for your team in that page (CTRL+F and then type in your teams name or abbreviation) and look out the links to find the one you want to use.

Once you  find the one you want to use copy the part that is after the team abbreviation and the first underscore ("_") up to just before the .svg

Exemple: 
From this link:
https://assets.nhle.com/logos/nhl/svg/MTL_19191920-19201921_light.svg

you need: `19191920-19201921_light`

Now open this config file : `config/logos.json` and you should see this

```
{
  "_default": "light",
  "WSH": "alt",
}
```

add your team abbreviation and the part you copied from the link like so:

`  "MTL": "19191920-19201921_light" `

save and now the new logo should replace the original one

![20210920_201458](https://user-images.githubusercontent.com/16941896/134093491-1c489f5f-a25c-4aac-9226-7cdcc567f2ed.jpg)

**Comment by recorderDopy** [comment]
I really like the customization this adds.

Is there any way to change the logo's alignment? I tried out the Mighty Ducks logo and it looks great on the Team Summary board but can hardly be seen on the scoreticker.

**Comment by riffnshred** [comment]
Yup ! the config file for the logos position is under config/logos.json. 

create a copy of that file and name it `custom_logos.json` and modify it as you please. FYI, on updates, the logos.json is overwritten to what is in the master version, but the custom_logos.json is left untouched.

**Reply by recorderDopy** [reply]
Yeah I was able to copy the logos.json file and use custom_logos.json to set the alternative logo. 

Are you able to adjust the logo zoom or position from custom_logs.json file? I got it working by modifying the config/layout/logos_64x32.json file but I imagine this will get overwritten during an update.
